zetaclient/chains/ton/observer/inbound.go

```diff
@@ -84,51 +88,78 @@ func (ob *Observer) observeInbound(ctx context.Context) error {
 		return nil
 	case len(txs) > MaxTransactionsPerTick:
 		ob.Logger().Inbound.Info().
-			Msgf("observeInbound: got %d transactions. Taking first %d", len(txs), MaxTransactionsPerTick)
+			Msgf("observeGateway: got %d transactions. Taking first %d", len(txs), MaxTransactionsPerTick)
 
 		txs = txs[:MaxTransactionsPerTick]
 	default:
-		ob.Logger().Inbound.Info().Msgf("observeInbound: got %d transactions", len(txs))
+		ob.Logger().Inbound.Info().Msgf("observeGateway: got %d transactions", len(txs))
 	}
 
 	for i := range txs {
-		tx := txs[i]
-
-		parsedTX, skip, err := ob.gateway.ParseAndFilter(tx, toncontracts.FilterInbounds)
-		if err != nil {
-			return errors.Wrap(err, "unable to parse and filter tx")
+		var skip bool
+
+		tx, err := ob.gateway.ParseTransaction(txs[i])
+		switch {
+		case errors.Is(err, toncontracts.ErrParse) || errors.Is(err, toncontracts.ErrUnknownOp):
```

Comment:
Can we maybe add context in comment here ?

Below error says `unable to parse tx` but here we skip for `ErrParse`
Might make sense to make error clearer